Resources/config/orm.xml

```diff
@@ -235,9 +235,9 @@
         </service>
 
         <service id="Doctrine\Bundle\DoctrineBundle\Controller\ProfilerController">
-            <argument type="service" id="twig" on-invalid="ignore"/>
```

Comment:
Instead of this whole PR, isn't proper fix to use `on-invalid="ignore_uninitialized"` instead?

Reply:
This indeed makes the container linter happy :thinking: 

But technically the service definition is still useless/invalid if `twig` or `profiler` are not available? I don't mind :blush: Can also adjust it.

Reply:
It's symfony's job to remove useless services automatically. If it doesn't do that, that would be an issue for https://github.com/symfony/symfony. symfony/dependency-injection shouldn't shift such responsibility to all the bundles for such use cases instead.

Reply:
Seems like this was discussed a while back: https://github.com/symfony/symfony/issues/36701

Anyway let's go with the simple solution for now. I will adjust the PR.